    [PATCH] VFS: local denial-of-service with file leases
    
    Remove time_out_leases() printk that's easily triggered by users.
    
    Signed-off-by: Chris Wright <chrisw@osdl.org>
    Signed-off-by: Trond Myklebust <Trond.Myklebust@netapp.com>
    
    Above is the signoff information for
    f3a9388e4ebea57583272007311fffa26ebbb305 included in Linus's 2.6 tree.
    As this is CVE-2005-3857 I am proposing its inclusion in 2.4
    
    Signed-off-by: Horms <horms@verge.net.au>
    
diff --git a/fs/locks.c b/fs/locks.c
index 370ed4c..2f21d25 100644
--- a/fs/locks.c
+++ b/fs/locks.c
@@ -1087,7 +1087,6 @@ static void time_out_leases(struct inode
 			before = &fl->fl_next;
 			continue;
 		}
-		printk(KERN_INFO "lease broken - owner pid = %d\n", fl->fl_pid);
 		lease_modify(before, fl->fl_type & ~F_INPROGRESS);
 		if (fl == *before)	/* lease_modify may have freed fl */
 			before = &fl->fl_next;
